portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentAPI.macro

```diff
@@ -89,7 +89,7 @@ definition {
 	}
 
 	macro _createStructuredContentDraft {
-		Variables.assertDefined(parameterList = "${data},${label},${name},${ddmStructureId},${title}");
+		Variables.assertDefined(parameterList = "${data},${label},${name},${ddmStructureId},${priority},${title}");
 
```

Comment:
the `priority` is an optional value, if we make it `Variables.assertDefined` then we will have to add it everytime we want to create the StructuredContent

Reply:
nvm, ignore the comment :)